.travis.yml

```diff
@@ -1,26 +1,46 @@
 language: node_js
-node_js:
-- node
-after_success:
-- coveralls < coverage/lcov.info
-- npm run-s build
-- test $TRAVIS_BRANCH = "main" && cd example && npm install &&
-  npm run build && cd ..
-deploy:
-- provider: pages
-  local_dir: example/build
-  skip_cleanup: true
-  github_token: "$GITHUB_TOKEN"
-  keep_history: true
-  on:
-    branch: main
-- provider: npm
-  skip_cleanup: true
-  email: devs@ginkgobioworks.com
-  api_key: "$NPM_TOKEN"
-  on:
-    tags: true
-    repo: ginkgobioworks/react-json-schema-form-builder
+node_js: node
+stages:
+  - test
+  - deploy
+jobs:
+  include:
+    - stage: test
+      script: npm run-s test:lint
+    - script: npm run-s test:flow
+
+    - script: npm run-s test:unit
+      after_success: coveralls < coverage/lcov.info
+    - node_js: 14
+      script: npm run-s test:unit
+    - node_js: 12
+      script: npm run-s test:unit
+
```

Comment:
Why nothing against 13?

Reply:
Node13 is no longer supported. Only even numbered versions are long term supported.

Reply:
Ah, right.